build/android/src/main/java/net.minetest.minetest/MtNativeActivity.java

```diff
@@ -19,6 +19,10 @@ public void onCreate(Bundle savedInstanceState) {
 	public void onDestroy() {
 		super.onDestroy();
 	}
+	
+	@Override
+	public void onBackPressed() {
```

Comment:
Standard Android closes the application by pressing the back button if this is the first activity.